panel/src/panel/content.js

```diff
@@ -34,6 +34,18 @@ export default (panel) => {
 			return changes;
 		},
 
+		/**
+		 * Closes the retry dialog if it is still open
+		 */
+		closeRetryDialog() {
+			if (
+				panel.dialog.isOpen &&
+				panel.dialog.props.id === "content-retry-dialog"
+			) {
+				panel.dialog.close();
+			}
+		},
```

Comment:
`Thought`: I found the method not super elegant (without a openRetyDialog e.g.). But this is really nitpick and can be ignored if you disagree. Was thinking of instead adding a `dialog` property in the content module that is `null` by default but set to `panel.dialog` when the dialog was opened. Cause then one could always just to `panel.content.dialog?.close()`.

Reply:
I really liked the idea of the dialog prop. Already implemented :)